functests/utils/cluster/cluster.go

```diff
@@ -29,3 +33,29 @@ func ComputeTestTimeout(baseTimeout time.Duration, isSno bool) time.Duration {
 
 	return testTimeout
 }
+
+// EnforceRequirements indicates if tests with specific cluster preconditions should fail rather than skip
+// if the cluster we are running against doesn't meet the requirements of the tests
+func EnforceRequirements() bool {
```

Comment:
maybe we can use this in other tests (different PR)

Reply:
Yep, this is the plan